website/docs/r/serviceendpoint_gcp.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_serviceendpoint_gcp"
+description: |-
+  Manages a GCP service endpoint within Azure DevOps organization.
+---
+
+# azuredevops_serviceendpoint_gcp
+Manages a GCP service endpoint within Azure DevOps.
+
```

Comment:
GCP extension is not installed by default. Add note which extension need installed before using this resource.